shell/server/installer.js

```diff
@@ -385,6 +388,10 @@ AppInstaller = class AppInstaller {
             (manifest.appTitle && manifest.appTitle.defaultText), manifest.appVersion,
             (manifest.appMarketingVersion && manifest.appMarketingVersion.defaultText));
         }
+
+        if (_this.isPreinstalled) {
```

Comment:
What if someone upload-installs a non-public version of an update to an app marked as a preinstall app?

What if I have preinstalled apps, but don't want to update them on my account from where I'm at?